src/antlr4/Cypher.g4

```diff
@@ -51,10 +51,26 @@ kU_DropTable
 DROP : ( 'D' | 'd' ) ( 'R' | 'r' ) ( 'O' | 'o' ) ( 'P' | 'p' ) ;
 
 kU_AlterTable
-    : ALTER SP TABLE SP oC_SchemaName SP DROP SP oC_PropertyKeyName ;
+    : ALTER SP TABLE SP oC_SchemaName SP kU_AlterOptions ;
 
 ALTER: ( 'A' | 'a' ) ( 'L' | 'l' ) ( 'T' | 't' ) ( 'E' | 'e' ) ( 'R' | 'r' ) ;
 
+kU_AlterOptions
+    : kU_AddColumn
+        | kU_DropColumn;
+
+kU_AddColumn
+    : ADD SP ( COLUMN SP )? oC_PropertyKeyName SP kU_DataType ( SP DEFAULT SP oC_Expression )? ;
+
+DEFAULT : ( 'D' | 'd' ) ( 'E' | 'e' ) ( 'F' | 'f' ) ( 'A' | 'a' ) ( 'U' | 'u' ) ( 'L' | 'l' ) ( 'T' | 't' ) ;
+
+kU_DropColumn
+    : DROP SP ( COLUMN SP )? oC_PropertyKeyName ;
+
+ADD: ( 'A' | 'a' ) ( 'D' | 'd' ) ( 'D' | 'd' ) ;
+
+COLUMN: ( 'C' | 'c' ) ( 'O' | 'o' ) ( 'L' | 'l' ) ( 'U' | 'u'  ) ( 'M' | 'm' ) ( 'N' | 'n' ) ;
```

Comment:
remove if you decide to remove  (COLUMN SP)?

Reply:
since we are following duckdb's grammar , we should give the user flexibility to omit the `column` keyword.